brainscore_language/submission/endpoints.py

```diff
@@ -1,35 +1,13 @@
-import argparse
-import os
-import requests
-from requests.auth import HTTPBasicAuth
 from typing import List, Union, Dict
 
 from brainscore_core import Score, Benchmark
-from brainscore_core.submission import UserManager, RunScoringEndpoint, DomainPlugins
+from brainscore_core.submission import RunScoringEndpoint, DomainPlugins
+from brainscore_core.submission.endpoints import make_argparser, retrieve_models_and_benchmarks, get_user_id, \
```

Comment:
```suggestion
from brainscore_core.submission.endpoints import make_argparser, get_models_and_benchmarks, get_user_id, \
```

Reply:
it's called `retrieve` in https://github.com/brain-score/core/pull/50/files#diff-506f4072002058e6ccf167d426583df7e54c7fe005bc22c6bac28c610c2cf71fR263, but can change if you prefer